cpp/system_updater.cc

```diff
@@ -131,9 +131,17 @@ int Main(int argc, char *argv[]) {
 
     std::sort(script_names.begin(), script_names.end(), ScriptLessThan);
     std::string last_schema, current_schema;
+    int previous_version_number = -1;
 
     for (const auto &script_name : script_names) {
         LOG_INFO("Running " + script_name);
+        const unsigned version_number(GetVersionFromScriptName(script_name));
+
+        if (previous_version_number == (int) version_number)
```

Comment:
Please use StringUtil::ToInt instead

Reply:
I don't think using StringUtil::ToInt will work as that is for converting std::string to int!  What would be better is to use static_cast but that will have the same effect.